src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Mobile/MyExtensionsApp.Mobile.csproj

```diff
@@ -92,7 +92,7 @@
 	</ItemGroup>
 	<!--#endif-->
 	<Choose>
-		<!--#if (android)-->
+		<!--#if (useAndroid)-->
```

Comment:
I'm not convinced commenting these out is a good idea even if the platform isn't selected - it just makes it very hard/painful to add platforms later.